cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterDescriptor.java

```diff
@@ -152,6 +152,34 @@ public void loadProps() {
           .parseInt(properties.getProperty("read_data_consistency_level",
               Integer.toString(conf.getReadDataConsistencyLevel()))));
 
+      conf.setConcurrentQPSubTaskThread(Integer
+          .parseInt(properties.getProperty("concurrent_qp_sub_task_thread",
+              Integer.toString(conf.getConcurrentQPSubTaskThread()))));
+      if (conf.getConcurrentQPSubTaskThread() <= 0) {
+        conf.setConcurrentQPSubTaskThread(Runtime.getRuntime().availableProcessors());
```

Comment:
default value in config is Runtime.getRuntime().availableProcessors() * 10, here is Runtime.getRuntime().availableProcessors()?

Reply:
fixed